src/+otp/+utils/+compatibility/odextend.m

```diff
@@ -0,0 +1,44 @@
+function solExtend = odextend(sol, f, tfinal, y0, opts)
+% OCTAVE FIX: naive odextend implementation to fix event function
```

Comment:
Oh wow, nice fix for my naive implementation. I know you didn't want to implement this, but I feel like having all problems work with octave would be super nice.

Very nice.